client/src/sections/Login/Login.tsx

```diff
@@ -1,17 +1,27 @@
-import React from 'react';
+import React, {useState, useCallback} from 'react';
 import {useHistory} from 'react-router-dom';
 import {Button, Card, Heading, ButtonGroup} from '@shopify/polaris';
-import logo from 'icons/logo.jpg';
+import logo from 'icons/logo.png';
 
 import './Login.css';
+import LoginModal from './LoginModal';
```

Comment:
Let's try and keep our project structure consistent:
* Make an `index.ts` file in the same directory as `<LoginModal />` that exports this component by default
* Make a `component.ts` file in the directory of `<Login />` that exports default as LoginModal

There are lots of examples in our project to use as references. If you can't figure it out let me know right away.